pages/pipelines/build_retention.md

```diff
@@ -50,6 +50,8 @@ The following diagram shows the lifecycle of build data by plan.
   </tbody>
 </table>
 
+Per pipeline retention settings have been retired and all pipelines are now in organisation specific retention settings specified above.
```

Comment:
This currently reads like a changelog, but the docs should reflect the current state. How about:

```suggestion
Retention periods are set according to an organization's plan, as shown in the previous table. Per-pipeline retention settings are not supported.
```

Does that work?